invenio_rdm_records/records/api.py

```diff
@@ -264,6 +275,17 @@ class RDMDraft(CommonFieldsMixin, Draft):
         delete=False,
     )
 
+    media_files = FilesField(
```

Comment:
having said: https://github.com/inveniosoftware/invenio-rdm-records/pull/1335/files#r1247621839
we still need to have the api class providing the media_files field - otherwise it won't be dumped in the record's payload on `/api/record/<recid>`